### DISCUSSION TITLE
Version supported for Tomcat and Java

### DISCUSSION THREAD

**Discussion by chintan-vaghani2008** [open-telemetry/opentelemetry-java-contrib#1363] [Q&A]
Can we know which are the version supported by this library for Apache Tomcat and Java ?

**Comment by trask** [comment]
are you asking about the opentelemetry java agent, or something else? for the java agent, check out https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#application-servers

**Reply by Ifedayo123** [reply]
We needed the answer
